rust-bindings/src/kernel_args.rs

```diff
@@ -47,6 +64,15 @@ impl KernelArgs {
     }
 
     /// Append the entire contents of the currently booted kernel commandline.
+    // rustdoc-stripper-ignore-next-stop
+    /// Appends the command line arguments in the file "/proc/cmdline"
```

Comment:
cool that the C docs are copied to Rust now

Reply:
I think probably this happened because I did : `$ make merge-lgpl-docs` which https://github.com/ostreedev/ostree/blob/main/rust-bindings/README.md#documentation says I should not have done